data-sources/discord/output/discord-usernames.csv

```diff
@@ -1,263 +1,263 @@
+51,0x36e9a9184787346215f8b41cc4c1cf75c3df14ce,,
+52,0x978452c747ee0b617285caf3c50cae0103af5656,,
+53,0x5a6d32dd7f169fae453e0f0866e5b8334f66889a,,
+54,0x186bad94057591918c3265c4ddb12874324be8ac,,
+55,0x3c9cfef0f5bbc4b1936bb1785ec485e56ca6f6ba,,
+56,0xf4b2eca9c0e8f40c70ea1dc5a6076891cafd3f10,,
+57,0xa67b181369939629495d6c36750ba5e8d7a62323,,
+58,0x5f33734d47761b1578f20808714adf2a669d23a8,,
+59,0x0d75363981488f3376fdce70fb5a6dd47c1ed935,,Adelaide Isla#3410
+60,0xb879be7af16a8c721ba381487627488b14519abc,,
+61,0x9fb203e1542a488f685f0085100660c7273cfd5b,,
+62,0x78b027de4c6211e0f958c986135fefcc31b7c4a1,,
+63,0x7a43c0f2f3f7ec470d200295f72292c98fd55307,,
+64,0x5e1c3834d5200fec3e01a51ccb70f6f00a2418aa,,
+65,0xe25b7ef2ebf5fc9826229533146f3817422f3b67,,
+66,0x503ffec4cb59b4d8cd2308fc793f6abce32cef0a,,
+67,0xca933e8a503e33be92a209d3abdee69ee747f019,,
+68,0xc173d302bd62b45754dc0abfad1407dddb40d050,,
+69,0x39a3fba9bfdd5f0fc7e83a9cd32c9fa425a40c80,,
+70,0x9151d344acab2dd9ccc40ae15b2963148edf18ee,,
+71,0xc9fcb06bf27293b10303b00ffa05adb63bf9b73b,,
+72,0xe28b3058a2f7f251741a7289b633a902126260ea,,
+73,0xea304a0d0e23a2bc5ffc48c8e841c11d71687c8c,,
+74,0xaa4197c2efcf7cc9919d582c9246237ebc4209d2,,
+75,0x2dea8f1662a0a64058971b4b9eb0a2fbe1dfc1e6,,
+76,0xd1e8487e49cc24a4879dcabbc946ab99489fa997,,
+77,0x29a4cf905fd5b4c24e488c25bc02078e08ba2268,,
+78,0x8c14d46c920c877d296aaf34f5fc65ff6d8c01cf,,
+79,0x01b009add41b107411266248ece40b0f3f98a413,,
+80,0x29fe8aa2798116332bc9a9dfac9a0072ad7c4196,,
+81,0x751893105b1ece3b7feda367b92cd1bcf2abd673,,
+82,0x1c208fcb9d15e0e04a0a579e0f6121e2057ff7f6,,
+83,0xcbd0b5491d4fd378d8a3261bfd8f321af63659f6,,
+84,0xc10c4dd2b2d51c575810beef45a033c96a3be1ba,,
+85,0x1fc1d46d195a8649adb5188a2506fa62e07d0654,,
+86,0xabf26352aadaaa1cabffb3a55e378bac6bf15791,,
+87,0x36aa46e405d2a605cb0bd6b15476fcb2a3dfc1b2,,
+88,0x701d7ec468a8ff9bd4b76be049e40cab79fb3310,,
+89,0x5df52e6b70f25919ad29add390efe2614f91b2c6,,
+90,0xc8d4a8970035f9f3fd01dbd238c97b5348aa3199,,
+91,0x6b2b69c6e5490be701abfbfa440174f808c1a33b,,
+92,0xbfa43bf6e9fb6d5cc253ff23c31f2b86a739bb98,,
+93,0x25ad71e78af2b33b3aeb2573c1df93aa078a6216,,
+94,0xc6034fb18abb066605c5d9a45039664f2b95eeb0,,
+95,0xdc260da83cd84772e3bc12de2066aff673e0b654,,
+96,0x0b42185d8b613af51ce1bf1e0992dd00ef893089,,
+97,0x39978cc40e2d1d7e127050bdfffbb0dfcfaebad0,,
+98,0x61751bc401f4eabe10a238661b4a60ba84910059,,
+99,0x8e9f36ce580ca12d96e112aa1ad5058610ce752f,,
+100,0xcf2b7c6bc98bfe0d6138a25a3b6162b51f75e05d,,
+101,0xaf4807d083287f205d897e6d00c6fde1bf0a241a,,
+102,0xa51b7d35376809156528108e6c2e2f200a24fd42,,
+103,0x42b39b6a479159749ab9b5c6681db69d41b12a52,,
+104,0x78336f51daa5cfd03a1c3e4a76f88cc0818a60d9,,
+105,0xb23e8c29a9cc468ba60e0438233398bdb64a6897,,
+106,0xddf767f258adf0af89896621349cadcf8722f771,,
+107,0xeb8b597fbed029083dfa2c449e32d9560e8591d3,,
+108,0xd15b5fa5370f0c3cc068f107b7691e6dab678799,,
+109,0x2337f09ef3192aebd8477cdc34d029804f7edb92,,
+110,0x8c9c8a244c71a23b45ebbdd9d881afe993d5b13d,,https://discord.gg/grbsArn
```

Comment:
@johnmark13 Since people can put _any_ value in an ENS text record, we might have to use a reg exp expecting `User Name#1234`?